project/Dependencies.scala

```diff
@@ -16,12 +16,13 @@ object Dependencies {
   object Versions {
     val akka = sys.props.getOrElse("build.akka.version", "2.6.10")
     val alpakka = "2.0.2"
-    val alpakkaKafka = sys.props.getOrElse("build.alpakka.kafka.version", "2.0.5")
+    val alpakkaKafka = sys.props.getOrElse("build.alpakka.kafka.version", "2.1.0-M1")
```

Comment:
@seglo, is it fine to upgrade alpakka kafka (major version) on projections? 

I'm fine with that, but wondering how this could impact users directly.

Reply:
The major change is that we drop support for Akka 2.5, Scala 2.11, and our testkit embedded-kafka support. Projections users are using akka 2.6 from the start and the other things are irrelevant, so I don't think it should be a problem.

Reply:
While this will work for Akka Projection users, it sends a bad signal to depend on a milestone version for a non-milestone release.

Reply:
Yes, we should wait for a final artifact before release.